src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -56,7 +68,9 @@ baseDaoContract config@Config{..} = defaultContract $ contractName cDaoName $ do
 fa2Handler
   :: (IsoValue pm, HasFuncContext s (BaseDAO.Storage pm))
   => Entrypoint' FA2.Parameter (BaseDAO.Storage pm) s
-fa2Handler =
+fa2Handler = do
+  docStorage @(BaseDAO.Storage pm)
```

Comment:
Why is storage description here?

(not sure if @Martoon-00's comment applies to this line as well)